### PR TITLE
Resto druid - Added high initial overhealing WG suggestions and checklist

### PR DESCRIPTION
**Pull request by buimichael**
[Screenshot from Gyazo](https://gyazo.com/167518355e067cc83a597b36ba066830)
[Screenshot from Gyazo](https://gyazo.com/7f9f46b05145db63dde2ec1359c6078e)

### CODE REVIEW

File: src/parser/druid/restoration/modules/features/Checklist/Component.js [hunk 1]
```diff
@@ -53,22 +53,30 @@ class RestorationDruidChecklist extends React.PureComponent {
...
             )}
             tooltip="This is your percent of Wild Growth casts that hit too few wounded targets. Low target casts happen either by casting it when almost all the raid was full health, or casting it on an isolated target. Remember that Wild Growth can only apply to players within 30 yds of the primary target, so if you use it on a target far away from the rest of the raid your cast will not be effective."
             thresholds={thresholds.wildGrowthPercentBelowRecommendedCasts} />
+          <Requirement
+            name={(
+              <>
+                High initial overhealing <SpellLink id={SPELLS.WILD_GROWTH.id} />
+              </>
+            )}
+            tooltip="This is your percent of Wild Growth casts that has high initial overhealing. Wild Growh does most of it's healing initially and declines over duration. Make sure you are not precasting it before damaging event but after damage occurs."
```
Comment (awlego) [inline src/parser/druid/restoration/modules/features/Checklist/Component.js]: Spelling mistake of Wild Growth at the beginning of the second sentence.

File: src/parser/druid/restoration/modules/features/WildGrowth.js [hunk 2]
```diff
@@ -68,6 +115,10 @@ class WildGrowth extends Analyzer {
     return (this.belowRecommendedCasts / this.wgs) || 0;
   }
 
+  get percentBelowRecommendedCasts2() {
```
Comment (awlego) [inline src/parser/druid/restoration/modules/features/WildGrowth.js]: I would rename the functions that have '2' in the name to include something more indicative about what the recommendation actually is (in this case it appears to be bad pre-casts or wasted initial WG tick healing). Just a suggestion -- not a requirement.

Comment (buimichael) [reply src/parser/druid/restoration/modules/features/WildGrowth.js]: Thanks, was being lazy at the time. :)

File: src/parser/druid/restoration/modules/features/Checklist/Component.js (hunk 1 above)
Comment (buimichael) [reply src/parser/druid/restoration/modules/features/Checklist/Component.js]: :+1:

**Comment by awlego** [comment]
Looks good to me -- but you'll probably have to get an actual reviewer to do an official review.

**Pull request merged**